README.md

```diff
@@ -48,10 +48,11 @@ Notice **there are no generics, and no manual typing.** Your endpoint’s exact
 
 ## 🔧 Setup
 
-First install this package and [openapi-typescript](https://github.com/drwpow/openapi-typescript) from npm:
+First, install this package and [openapi-typescript](https://github.com/drwpow/openapi-typescript) from NPM:
```

Comment:
```suggestion
First, install this package and [openapi-typescript](https://github.com/drwpow/openapi-typescript) from npm:
```

`npm` is actually the correct capitalization. It is no longer an acronym, and they have been very clear. Comma is a fine change.

Reply:
Didn’t know that, thank you!